aif360/sklearn/explainers/bias_explainer.py

```diff
@@ -0,0 +1,193 @@
+from fairxplainer.fair_explainer import FairXplainer
+from fairxplainer.fair_explainer import plot as fif_plot
+import numpy as np
+
+
+def explain_statistical_parity(
+        clf, 
+        X, 
+        sensitive_features, 
+        maxorder=2, 
+        spline_intervals=5,
+        k=7,
+        seed=22, 
+        cpu_time=300, 
+        verbose=False):
+    """
+        Explain the statistical parity of a classifier given a dataset
+
+    Args:
+        clf: the trained model to explain
+            clf.predict() should be implemented
+        X (pandas.DataFrame): the dataset (containing the features) to measure the bias on
+        sensitive_features (list): list of sensitive attributes
```

Comment:
can this be renamed `prot_attr` for consistency with metrics, etc.?

Reply:
Sure. I am addressing this.